pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -330,31 +325,105 @@ func (c *Reconciler) reconcile(ctx context.Context, tr *v1alpha1.TaskRun) error
 	return nil
 }
 
-func updateStatusFromBuildStatus(taskRun *v1alpha1.TaskRun, buildStatus buildv1alpha1.BuildStatus) {
-	if buildStatus.GetCondition(duckv1alpha1.ConditionSucceeded) != nil {
-		taskRun.Status.SetCondition(buildStatus.GetCondition(duckv1alpha1.ConditionSucceeded))
-	} else {
-		// If the buildStatus doesn't exist yet, it's because we just started running
+func updateStatusFromPod(taskRun *v1alpha1.TaskRun, pod *corev1.Pod) {
+	fmt.Println("taskrun.Status", taskRun.Status)
+	fmt.Println("taskrun.Status.ConditionSucceed", taskRun.Status.GetCondition(duckv1alpha1.ConditionSucceeded))
```

Comment:
these lines might have been included by accident? :innocent:

Reply:
ahh damn I forgot to remove it :sweat_smile: 
#jedi-power-you-never-saw-those-lines